doc/sphinx/source/esmvalcore/preprocessor.inc

```diff
@@ -107,32 +290,41 @@ See also :func:`esmvalcore.preprocessor.seasonal_mean`.
 6. annual_mean
 --------------
 
-This function produces an annual or a decadal mean. The only argument is the
-decadal boolean switch. When this switch is set to true, this function
-will output the decadal averages.
+This function produces an annual or a decadal mean. It takes a single boolean
+switch as argument:
+* decadal: calculate decadal averages.
 
 See also :func:`esmvalcore.preprocessor.annual_mean`.
 
 
 7. regrid_time
 --------------
 
-This function aligns the time points of each component dataset so that the dataset
-iris cubes can be subtracted. The operation makes the datasets time points common and
-sets common calendars; it also resets the time bounds and auxiliary coordinates to
-reflect the artifically shifted time points. Current implementation for monthly
-and daily data; the frequency is set automatically from the variable CMOR table
-unless a custom frequency is set manually by the user in recipe.
+This function aligns the time points of each component dataset so that the
+dataset Iris cubes can be subtracted. The operation makes the datasets time
+points common and sets common calendars; it also resets the time bounds and
+auxiliary coordinates to reflect the artifically shifted time points. Current
+implementation for monthly and daily data; the frequency is set automatically
+from the variable CMOR table unless a custom frequency is set manually by the
+user in recipe.
```

Comment:
And this is also a little vague, the custom frequency definition in the recipe. Could there be an example added?

Reply:
@ledm can you add an example?

Reply:
```suggestion
         
```
Not sure about this. Possibly best to remove this phrase as I don't see the functionality to add custom frequencies.